test/unit/facts.json

```diff
@@ -88,7 +88,7 @@
     "lsbmajdistrelease": "6",
     "swapfree": "16.72 GB",
     "netmask": "255.255.255.192",
-    "discovery_bootif": "AA:BB:CC:DD:EE:FF",
+    "discovery_bootif": "AA:BB:CC:DD:EE:F5",
```

Comment:
yes, many tests where failing because of mac turning into nil because of https://github.com/theforeman/foreman/pull/1923/files#r22298588
the "next mac" was trying to be created and since this one ends with `ff` it would be an invalid mac